apex/functions/sign-xpi/function.system.dev.json

```diff
@@ -0,0 +1,15 @@
+{
+  "runtime": "python",
+  "description": "The sign-xpi lambda with a set of credentials for system addons",
+  "timeout": 30,
+  "hooks": {
+    "build": "pip install -r requirements.txt -t site-packages"
+  },
+  "environment": {
+    "autograph_serverUrl": "http://52.91.167.168:8000/",
+    "autograph_hawkId": "alice",
+    "autograph_hawkSecret": "fs5wgcer9qj819kfptdlp8gm227ewxnzvsuj9ztycsx08hfhzu",
```

Comment:
I think this dev version of autograph hasn't been updated yet with different credentials for the two signers. I don't think it will be important for development of the lambda.